InternxtDesktop/Services/Auth/AuthManager.swift

```diff
@@ -148,6 +148,26 @@ class AuthManager: ObservableObject {
         return true
     }
     
+    func needRefreshToken() throws -> Bool {
+        guard let legacyAuthToken = config.getLegacyAuthToken() else {
+            throw AuthError.LegacyAuthTokenNotInConfig
+        }
+
+        let responseDecodeLegacy = try JWTDecoder.decode(jwtToken: legacyAuthToken)
+        
+        guard let expTimestamp = responseDecodeLegacy["exp"] as? Int else {
+            throw AuthError.InvalidTokenExp
+        }
+        
+        let expDate = Date(timeIntervalSince1970: TimeInterval(expTimestamp))
+        let currentDate = Date()
+
+        guard let daysUntilExpiration = currentDate.daysUntil(expDate) else {
+            return true
+        }
+
+        return daysUntilExpiration <= 3
```

Comment:
Move the "3" to a variable somewhere, and call it "REFRESH_TOKEN_DEADLINE" or something like that, this way we communicate the purpose of the value we are using here

Reply:
done!